src/bank.rs

```diff
@@ -774,8 +812,16 @@ mod tests {
 
     #[test]
     fn test_process_ledger_around_window_size() {
+        // benchmark
```

Comment:
How long does this take to execute? I run the few test suite constantly when developing via `cargo watch 'test --libs'` and want that instantaneous feedback. If this takes more than a second to execute, I'd prefer you move in to a `#[bench]`.

Reply:
I don't want to land the benchmark, it takes *awhile*

Reply:
Overall developer productively is a higher priority than fullnode startup time.